### ISSUE TITLE
Passing parent attributes to child when using <content>

### ISSUE THREAD

**Issue by bennyschudel** [vuejs/Discussion#276]
Hi

Is there a simple solution to pass certain parent properties to child when using the <content> tag?

Example:
```javascript

<script type="text/x-template" id="parent-component">
  ...
  <content>
  ...
</script>

<parent-component prop="true">
  <child-component prop-new="{{prop}}"></child-component>
</parent-component>
```

doesn't work for me.

**Comment by swift1** [comment]
This might help:
http://jsfiddle.net/3qtyhu75/

**Comment by swift1** [comment]
I think I figured out exactly what you're looking for. Check this:

**Comment by bennyschudel** [comment]
Hey thank you.

I don't want my component to inherit all the properties but just a specific one.

At the moment I've solved my issue by using an event but I thought that there is maybe a way to pass properties to a child.

**Comment by swift1** [comment]
If you only want to inherit specific properties, you can pass them as props, like this:
http://jsfiddle.net/1tv62ohp/1/

**Comment by bennyschudel** [comment]
Yes, that works. But my problem is that it does not work with the content tag

See here.

http://jsfiddle.net/npq05d8d/

**Comment by swift1** [comment]
You're using the content tag the wrong way. Check out my epic new improvement of the router and you will see how you can pass parameters to children: http://jsfiddle.net/zedtc2c4/ :smiley:
